src/components/Sections/SectionsList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
Yes. Good idea! That would make simpler for us to repeat naming in case we do something similar to Review but in its own Sections folder to avoid logic too complex by mixing both together. I'll move this now.

Reply:
Done: 7a92db3 & 83362c9